e2e-tests/web/test/pageobjects/dashboard.page.ts

```diff
@@ -0,0 +1,16 @@
+import Page from './page'
+
+class DashboardPage extends Page {
+  public language: string
+
+  public constructor(language = 'en') {
+    super()
+    this.language = language
+  }
+
+  get searchIcon() {
+    return $('//header/div/div[4]/div[1]/a')
```

Comment:
@Taggotty is this fine or is there a better way to get this icon. I tried to use Appium Image selector, but could not get this working.
https://webdriver.io/docs/selectors/#appium-image-selector

Reply:
it makes sense that the Appium Image selectors do not work, as you are writing web e2e tests. Appium is the native e2e backend. For web you can use this query: `//header//a[@aria-label='Suchen']`. For xPath queries open the browser navigate to the page where you need to query and you can test your xPath query with x('<your xPathQuery') -> https://devhints.io/xpath

Reply:
Thanks for explaining :)
Fixed it and now it's a lot nicer :+1: